js/graviex.js

```diff
@@ -0,0 +1,991 @@
+                        'deposit_address', // Requires access_key, tonce, signature, currency and api secret.
+                        'orders', // Requires access_key, tonce, signature and api secret. Optional market, state, limit, page, order_by
+                        'order', // Requires access_key, tonce, signature and api secret AND id (unique orderid).
+                        'trades/my', // Requires market XXXYYY, access_key, tonce, signature and api secret . Optional: limit, timestamp, from, to, order_by
+                        'trades/history', // Requires market XXXYYY, access_key, tonce, signature and api secret . Optional: limit, timestamp, from, to, order_by
+                        'orders/history', // Requires market XXXYYY, access_key, tonce, signature and api secret . Optional: limit, timestamp, from, to, order_by
+                        'gen_deposit_address', // Requires access_key, tonce, signature, currency and api secret.
+                        'account/settings', // Requires access_key, tonce, signature . Not Using Currently In Unified Methods.
+                        'fund_sources', // Requires access_key, tonce, signature . Not Using Currently In Unified Methods.
+                        'strategies/list', // Not Using Currently In Unified Methods.
+                        'strategies/my', // Not Using Currently In Unified Methods.
+                    ],
+                    'post': [
+                        'orders', // Requires access_key, tonce, signature and api secret AND market, side(sell/buy), volume. Optional price ord_type
+                        'orders/multi', // Requires access_key, tonce, signature and api secret AND market, orders, orders[side(sell/buy)], orders[volume]. Optional orders[price], orders[ord_type]. To create multiple orders at once
+                        'orders/clear', // Requires access_key, tonce, signature and api secret. Optional side(sell/buy) to only clear orders on one side.
+                        'order/delete', // Requires access_key, tonce, signature and api secret AND id (unique orderid).
+                        'account/store', // Not Using Currently In Unified Methods.
+                        'create_fund_source', // Not Using Currently In Unified Methods.
+                        'remove_funs_source', // Not Using Currently In Unified Methods.
+                        'strategy/cancel', // Not Using Currently In Unified Methods.
+                        'strategy/create', // Not Using Currently In Unified Methods.
+                        'strategy/update', // Not Using Currently In Unified Methods.
+                        'strategy/activate', // Not Using Currently In Unified Methods.
+                        'strategy/deactivate', // Not Using Currently In Unified Methods.
+                    ],
+                },
+            },
+            'fees': {
+            },
+            'limits': {
+                'amount': {
+                    'min': 0.001,
```

Comment:
this looks a bit suspicious, why are the amount limits hardcoded

Reply:
@frosty00, Shall we remove this as amount limits and precision object values are not in use ?

Reply:
yeah i'd remove this, it should be inside of `fetchMarkets`

Reply:
actually it looks ok, we can keep it